www/index.html

```diff
@@ -17,6 +17,10 @@
     SlabNum = "";
     while (SlabNum == "" || SlabNum == null || SlabNum == 0) {
       SlabNum = prompt("Enter number of Slabs:", "");
+      if (isNaN(SlabNum) == true) {
```

Comment:
Can we do similar form/input validations for all inputs?

Only if it's not too much to ask (optional).

Reply:
we have following input types : 
1. AI NAME : characters and integers are accepted. 
2. IntName : characters and integers are accepted. 
3. Slab Count : Only Numbers are accepted [Check is there.]
4. Slab Name : characters and integers are accepted. 
5. Justification : characters and integers are accepted. 
6. Transactions : characters and integers are accepted.
7. Quantum Score : Only 0 and 1 are accepted [Check is there]

I did't put check for remaining types thinking - input could be of any kind, for example : Slab name could  be 001, 002 for a particular AI model . For justification and transaction user is free to use both numbers and strings. 

Kindly let me know any case which i might be missing ,i will take care of it.

Reply:
Yes @pankaj-nayak, it could be an alphanumeric string.